gfootball/env/observation_processor.py

```diff
@@ -138,6 +171,34 @@ def softmax(x):
   return np.exp(x) / np.sum(np.exp(x), axis=0)
 
 
+def pprint_players_actions(writer, players_actions):
+    table_text = [["TEAM", "PLAYER", "SPRINT", "DRIBBLE", "DIRECTION",
+                   "ACTION"]]
+    widths = [35, 50, 50, 55, 60, 50]
+
+    team_short_name = {'left': 'L',
+                       'right': 'R'}
+    direction_short_name = {'-': '-',
```

Comment:
This might be hard to read, could we add some sprites / unicodes like ←↖ etc (https://en.wikipedia.org/wiki/Arrow_(symbol))

Reply:
It couldn't be done to add arrow unicode via cv2.putText, so there is separate method for writing arrows